02-oop-inheritance/patterns-1.1.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+
+    <meta charset="utf-8">
+    <title>Patterns and OOP in JavaScript</title>
+
+    <script type="text/javascript" src="patterns-1.1.js"> </script>
+  </head>
+
+  <body>
+
+    <div class="container">
+
+        <input type="text" name="movieTitle" id="movieTitle">
```

Comment:
**Minor:** remember to include labels or add placeholders to each input, so that the user can know what goes on where